api/index.coffee

```diff
@@ -14,7 +14,7 @@ app.use cors()
 app.use helpers
 app.use bodyParser.json limit:'5mb', extended: true
 app.use bodyParser.urlencoded limit:'5mb', extended: true
-app.use morgan 'dev'
+app.use morgan(if NODE_ENV is 'development' then 'dev' else 'tiny')
```

Comment:
<3 thanks @kanaabe!  Outputting necessary logging keeps our syslog servers healthy and happy ^__^